src/test/objectstore/Fragmentation_simulator.cc

```diff
@@ -69,7 +72,7 @@ void FragmentationSimulator::init(const std::string &alloc_type, uint64_t size,
   std::cout << "Initializing ObjectStoreImitator" << std::endl;
   os = new ObjectStoreImitator(g_ceph_context, "", min_alloc_size);
 
-  std::cout << "Initializing allocator: " << alloc_type << " size: 0x"
```

Comment:
We have a controllable debug subsystem `dout(..)`.
It is better to have debugs present and ready to turn on when needed.
Please add new one, or use "test".
https://github.com/ceph/ceph/blob/main/src/common/subsys.h#L83